app/models/access_token.rb

```diff
@@ -36,8 +36,8 @@
 #
 
 class AccessToken < OauthToken
-  belongs_to :user
-  belongs_to :client_application
+  belongs_to :user, :optional => true
+  belongs_to :client_application, :optional => true
```

Comment:
Are these really optional? A token that isn't associated with a user or application?